Update notes.py

Yakiv2023 opened the pull request:
скрипт має дві опції (меню): 1. додати нотатку 2. вийти. Після виходу нотатки зберігаються у файлі notes.txt

Yakiv2023 committed:
Update notes.py

скрипт має дві опції (меню): 1. додати нотатку 2. вийти. Після виходу нотатки зберігаються у файлі notes.txt

VladyslavBon commented on personal_helper/personal_helper/notes/notes.py:
```diff
@@ -1 +1,29 @@
+        print('Нотатка успішно додана!')
+
+    elif choice == '2':
+        # Збереження нотаток у файл
```
Додай збереження нотатки у перший пункт, щоб вона додавалась і одразу зберігалась, так буде правильніше, другий пункт залиш просто для виходу

VladyslavBon commented on personal_helper/personal_helper/notes/notes.py:
```diff
@@ -1 +1,29 @@
+notes = []
```
Нотатка зберігається, але вона при кожному новому виході перезаписує себе, якщо щось записати і вийти, а потім запустити програму і знову записати щось або просто вийти, програма перезаписує файл, треба зробити щоб зберігалась і нова інформація і стара також

Yakiv2023 committed:
Create notes.py

Цей скрипт виконує дві опції: 1. додавання нотатку 2. вийти. Після виходу нотатки зберігаються у файлі notes.txt

Yakiv2023 committed:
Update notes.py

Yakiv2023 committed:
Update notes.py

Yakiv2023 committed:
Update notes.py

Yakiv2023 committed:
Update notes.py
